### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by rorysedgwick**

### CODE REVIEW

File: test/back/handlers.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
+var assert  = require("assert");
+var request = require("superagent");
```
Comment (MIJOTHY) [inline test/back/handlers.js]: what's this here for?

File: api/server.js [hunk 2]
```diff
@@ -55,19 +56,30 @@ server.route([
   },
 
   {
-    path    : "/jobs",
+    path    : "/{param*}",
```
Comment (MIJOTHY) [inline api/server.js]: lets get the routes into another file

File: test/back/handlers.js (hunk 1 above)
Comment (rorysedgwick) [inline test/back/handlers.js]: Thinking of using superagent before I realised server.inject was the way

File: api/server.js (hunk 2 above)
Comment (rorysedgwick) [inline api/server.js]: On it today

**Pull request merged**